src/reducers/searchParsers.js

```diff
@@ -4,6 +4,19 @@ const FILTER_ORDER = ['biotype', 'species'];
 import { makeFieldDisplayName } from '../lib/searchHelpers';
 import { NON_HIGHLIGHTED_FIELDS } from '../constants';
 
+export function flattenWithPrettyFieldNames(highlights) {
```

Comment:
You don't need to export this function unless you foresee using it in another module.